contrib/mergeBenchmarkSets.py

```diff
@@ -103,6 +103,12 @@ def get_witness_result(witness, verification_result):
             f"witness invalid ({status_from_verification})",
             result.CATEGORY_ERROR,
         )
+    # A mismatch of the witness type counts as error of the verifier.
+    if status_from_validation == "ERROR (unexpected witness type)":
+        return (
+            f"witness mismatch ({status_from_verification})",
+            result.CATEGORY_ERROR,
+        )
```

Comment:
This is code duplication. The if condition in line 101 should just be extended, maybe even checking if satus_from_validation is one of the elements of a set with the two strings inside.

Reply:
But the reported status is also different in these two cases: "witness invalid" vs. "witness mismatch". So I'm in favor of the original solution.

Reply:
Ok, let's keep then.

Reply:
@MartinSpiessl Would you approve?

Reply:
`witness mismatch` is weird, this is not informative to the reader. It is not clear what is mismatching. I would propose to have `witness type mismatch`, which would also be consistent with how we call this in the witness linter code now.